packages/mp/src/mp/validate/utils.py

```diff
@@ -91,8 +91,10 @@ def load_integration_def(integration_path: pathlib.Path) -> YamlFileContent:
         raise FatalValidationError(msg) from e
 
 
-def load_components_defs(integration_path: pathlib.Path) -> dict[str, list[YamlFileContent]]:
-    """Load all component's definition files, organized by component type.
+def load_components_defs(
+    integration_path: pathlib.Path, components: list[str]
```

Comment:
To make it a bit more clear:

```suggestion
    integration_path: pathlib.Path, *components: str
```

And then you can use it without wrapping the second argument in a list every time